odk1-src/form-functions.rst

```diff
@@ -0,0 +1,674 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Colores
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  arctan
+  arg
+  bday
+  bp
+  br
+  chkn
+  colores
+  fsh
+  gustan
+  jr
+  nodeset
+  prefs
+  seleccionados
+  te
+  tres
+  π
+  
+  
+	
+******************************
```

Comment:
I doubt this would be possible, but it'd be nice to see an alphabetical list of all the things in a table to make it glanceable.

Maybe a link to https://opendatakit.github.io/xforms-spec/#xpath-functions would be good enough?